internal/services/delegation_helpers.go

```diff
@@ -39,26 +39,22 @@ func (s *Service) registerUnbondingSpendNotification(
 		Index: 0, // unbonding tx has only 1 output
 	}
 
-	go func() {
+	s.wg.Go(func() {
```

Comment:
`conc.WaitGroup` is the same as running in separate goroutines + safer (it catches panics) + there is `sync.WaitGroup` under the hood which we will need to wait in future when I rewrite our pipeline logic